README.md

```diff
@@ -1,2 +1,16 @@
-# SOEN341
-SOEN 341 Amazon Store Clone Project Winter 2022
+# SOEN341 Winter 2022
+
+# Objective
+Develop a Amazon store web platform.
+
+# Project
+
+
+# Team Members
+-Gabriel Karras
```

Comment:
Need a space here to turn it into a bullet point

Reply:
Fixed with commit